### ISSUE TITLE
Travis job timeout

### ISSUE THREAD

**Issue by etienne-napoleone** [BuildOnViction/tomochain-v1#56]
It seems like the stage " Build and test" sometimes error because of the "output" timeout (when a job doesn't write anything to stdout for longer than 10m, the job is killed and marked as errored) .

This timout happen sometimes during the coverage test: `go run build/ci.go test -coverage`.
I believe this happen because those tests takes quite some time and depending on the load the Travis workers are experiencing, it can reach 10m without output.
This can probably be resolved by increasing the verbosity.

**Comment by ngtuna** [comment]
There is a work around to extend the timeout here: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received However it shouldn't be waiting too much

**Comment by etienne-napoleone** [comment]
```
ok  	github.com/ethereum/go-ethereum/swarm	0.029s	coverage: 8.9% of statements
ok  	github.com/ethereum/go-ethereum/swarm/api	0.092s	coverage: 63.6% of statements
ok  	github.com/ethereum/go-ethereum/swarm/api/client	0.104s	coverage: 70.8% of statements
ok  	github.com/ethereum/go-ethereum/swarm/api/http	0.103s	coverage: 48.6% of statements
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
The build has been terminated
```

```
ok  	github.com/ethereum/go-ethereum/internal/guide	10.784s	coverage: 0.0% of statements
ok  	github.com/ethereum/go-ethereum/internal/jsre	0.111s	coverage: 48.5% of statements
?   	github.com/ethereum/go-ethereum/internal/jsre/deps	[no test files]
?   	github.com/ethereum/go-ethereum/internal/web3ext	[no test files]
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
The build has been terminated
```
Seems like it happens randomly during coverage test. @ngtuna If you  think it's normal (maybe some folders need more time to be processed?), I can add the fix you linked earlier.

**Comment by etienne-napoleone** [comment]
Checking if `travis_wait` resolves the issue with test build [397288245](https://travis-ci.org/tomochain/tomochain/builds/397288245)

**Comment by etienne-napoleone** [comment]
Ok it works, the only small downside is that the logs of the coverage test is only available after it finished running. Which is really not important in our case.
Created PR #65
